ddd-kata.md

```diff
@@ -0,0 +1,94 @@
+| [Domain Message Flow](https://github.com/ddd-crew/domain-message-flow-modelling) | Developing loosely linked systems involves more than just neatly drawn boundaries. Interactions between bounded contexts must also be carefully described. A bounded context is a subsystem in a software architecture that is aligned with a certain domain. It may be built as a microservice or as a monolithic module. A Domain Message Flow Diagram (DMFD) is a basic visual representation of the flow of messages (commands, events, and inquiries) between actors, bounded contexts, and systems in a single scenario.                                                                                                                                                                                                                                            |
+| [Bounded Context Canvas](https://github.com/ddd-crew/bounded-context-canvas)     | The Bounded Context Canvas is a collaborative design tool for creating and documenting a single bounded context. The canvas walks you through the process of developing a bounded context by pushing you to examine and make decisions about essential design components such as nomenclature, responsibilities, public interface, and dependencies.                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| [Aggregate Canvas](https://github.com/ddd-crew/aggregate-design-canvas)          | The Aggregate Design Canvas is a modeling tool that should be used in conjunction with design-level domain modeling efforts. Eric Evans first described aggregates as a lifecycle pattern. We define aggregate as a network of objects that serves as a consistency boundary for our domain regulations. Depending on how the aggregate is designed, we can either enforce them (making them invariant) or be compelled to implement corrective measures. As a result, it is critical to carefully establish aggregate borders, as they influence the behaviors modeled inside our domain. The canvas features a suggested working order that aids in iteratively discussing various components of the overall design.                                                    |
+
+## The DDD Kata
+
+### The Deliverables
+
+* **Form a working group** (up to 8 Person). Declare one member a domain expert (maybe a substitute) and one member as Facilitator / Moderator
+* Perform a **Big Picture Event Storming** with your group in Mural or Miro
+* **Identify the bounded context and aggregates** within evaluated process in Mural or Miro
+* Establish the message flow from Bounded Context to Bounded Context using the **Domain Message Flow** Diagrams in Mural or Miro.
```

Comment:
I'm getting lazy right, so general comment: capitalization reads very German here and I would ditch the "Mural or Miro" and potentially put that in a step 0: decide on a modeling tool and get a template from xyz